inlang/source-code/sdk2/src/project/loadProjectFromDirectory.ts

```diff
@@ -18,15 +21,56 @@ export async function loadProjectFromDirectory(
 		"blob"
 	>
 ) {
-	const project = await loadProjectInMemory({
+	const tempProject = await loadProjectInMemory({
 		// pass common arguments to loadProjectInMemory
 		...args,
 		blob: await newProject(),
 	});
-	await copyFiles({ fs: args.fs, path: args.path, lix: project.lix });
+	await copyFiles({ fs: args.fs, path: args.path, lix: tempProject.lix });
```

Comment:
Why is the `tempProject` needed?

Reply:
see todo https://github.com/opral/monorepo/blob/38da990bc2567e19d276065c88a4d30bd30f5e0e/inlang/source-code/sdk2/src/project/loadProjectFromDirectory.ts#L31

Reply:
roger